package.json

```diff
@@ -681,7 +683,7 @@
     "has-ansi": "^3.0.0",
     "hdr-histogram-js": "^1.2.0",
     "he": "^1.2.0",
-    "highlight.js": "9.15.10",
+    "highlight.js": "^9.18.5",
```

Comment:
Did you decide no to bump to 10.2? Moving away from 9.x sounds like a good idea

![image](https://user-images.githubusercontent.com/209966/101420948-8d136900-38f3-11eb-840f-4f6d0aba07fc.png)

Reply:
Yeah, I was trying to avoid too many changes in this PR, as I don't own the code that consumes `highlight.js` and I'm not sure I fully understand the implications of the breaking changes. I just wanted to bump to 9.latest until we get a chance to do a major version upgrade. It's worth noting that EUI currently relies on 9.latest as well (as of last week).